src/main.vue

```diff
@@ -0,0 +1,418 @@
+<template>
+    <div id="kiwi-component-container">
+        <div id="kiwi-app"/>
+    </div>
+</template>
+<script>
```

Comment:
Almost all of main.vue is duplicated from main.js which is going to cause a fair bit of maintenance. Can we re-use something here?

With that, some parts of this code is not needed such as the loggers `window.href.location` checking

Reply:
This is done.